packages/typespec-vscode/src/extension.ts

```diff
@@ -213,3 +231,16 @@ function showStartUpMessages(stateManager: ExtensionStateManager) {
     stateManager.cleanUpStartUpMessage(workspaceFolder.uri.fsPath);
   });
 }
+
+async function showOpenApi3(docUri: vscode.Uri, context: vscode.ExtensionContext) {
```

Comment:
please move related code to vscode-cmd folder. create a sub folder if multiple files are needed

Reply:
moved this and `openapi3-preview.ts` to `vscode-command`